core/cafe_service.go

```diff
@@ -393,9 +393,51 @@ func (h *CafeService) notifyClient(pid peer.ID) error {
 }
 
 // sendCafeRequest sends an authenticated request, retrying once after a session refresh
-func (h *CafeService) sendCafeRequest(
-	cafe peer.ID, envFactory func(*pb.CafeSession) (*pb.Envelope, error)) (*pb.Envelope, error) {
+func (h *CafeService) sendCafeRequest(cafe peer.ID, envFactory func(*pb.CafeSession) (*pb.Envelope, error)) (*pb.Envelope, error) {
```

Comment:
This change breaks out HTTP based cafe requests from p2p based requests, which was/is useful for debugging (led me to the db connection limit issue):
- `service.sendCafeRequest`
- `service.sendCafeHTTPRequest`